google-apps-script.js

```diff
@@ -21,14 +25,23 @@ function doPost(e) {
   try {
     Logger.log(e); // the Google Script version of console.log see: Class Logger
     record_data(e);
-
-    var mailData = e.parameters; // just create a slightly nicer variable name for the data
+    
+    // shorter name for form data
+    var mailData = e.parameters;
+    
+    // names and order of form elements
+    var dataOrder = JSON.parse(e.parameters.formDataNameOrder);
+    
+    // determine recepient of the email
+    // if you have your email uncommented above, it uses that `TO_ADDRESS`
+    // otherwise, it defaults to the email provided by the form's data attribute
+    var sendEmailTo = (typeof TO_ADDRESS !== "undefined") ? TO_ADDRESS : mailData.formGoogleSendEmail;
```

Comment:
Yes, you started a discussion of this over in #5 but then the concern of how to prevent spam is mentioned there and in #79 

Is it worth the convenience? Not sure.

We could provide a flag instead, so that a data-attribute determines if an email is sent, but the email must be provided server-side. I could make this change easily. What should the default behavior be, without the data-attribute? To not send an email? Or to send one?